packages/velo-external-db-core/lib/service/data.spec.js

```diff
@@ -125,8 +131,8 @@ describe('Data Service', () => {
 
     test('aggregate api', async() => {
         driver.givenAggregateResult(ctx.entities, ctx.collectionName, ctx.filter, ctx.aggregation)
-        aggregationTransformer.stubIgnoreTransform(ctx.aggregation)
-        filterTransformer.stubIgnoreTransform(ctx.filter)
+        aggregationTransformer.givenAggregateTransformResult(ctx.aggregation)
+        filterTransformer.givenTransformResult(ctx.filter)
```

Comment:
naming can be better here, let's think of something that will better reflect what is happening in the test